DistFiles/factoryCollections/Sample Shells/Vaccinations/previewMode.css

```diff
@@ -1,4 +1,4 @@
-﻿@media screen {
+@media screen {
```

Comment:
Changes to this file do not appear to have anything to do with the userPrefs problem?